common-util/functions/errors.js

```diff
@@ -0,0 +1,46 @@
+import isObject from 'lodash/isObject';
+import { notifyError as notifyErrorFn } from '@autonolas/frontend-library';
+// create a function to specific error message based on error code
+export const notifyError = notifyErrorFn;
+const getErrorMessage = (error) => {
+  if (isObject(error)) {
+    if ((error?.message || '').includes('OwnerOnly')) {
+      return 'You are not the owner of the component/agent';
+    }
+
+    if ((error?.message || '').includes('WrongUnitId')) {
+      return 'Unit ID is not valid';
+    }
+
+    if ((error?.message || '').includes('ClaimIncentivesFailed')) {
+      return 'You do not have any rewards to claim';
+    }
+
+    if ((error?.message || '').includes('TransferFailed')) {
+      return 'Transfer failed';
+    }
+
+    if ((error?.message || '').includes('execution reverted')) {
+      return 'Nothing to claim for the connected wallet';
+    }
+
+    return error?.message || 'Some error occured';
```

Comment:
occured -> occurred, btw, very convenient VS code extension for such typos 

<img width="841" alt="image" src="https://github.com/valory-xyz/autonolas-tokenomics-frontend/assets/22725775/b0b74b00-3dd6-4552-bf41-81d44acb89df">